addon/webextension/background/senderror.js

```diff
@@ -92,20 +102,26 @@ this.senderror = (function() {
       return;
     }
     if (!Raven.isSetup()) {
-      Raven.config(dsn, {allowSecretKey: true}).install();
+      Raven.config(dsn, {allowSecretKey: true, dataCallback: normalizeURL}).install();
     }
-    let exception = new Error(e.message);
+    // NOTE: don't use e.message, as that is a localized string
+    let exception = new Error(e.popupMessage);
```

Comment:
Also I noticed the errors had no description, which I realize it because we're using popupMessage.  Example: https://sentry.prod.mozaws.net/operations/screenshots-addon-dev/issues/612365/